.github/workflows/e2e-test-bridge.yml

```diff
@@ -93,13 +93,18 @@ jobs:
               id: check_date
               run: |
                 date --version
+            - name: Check output
+              id: check_output
+              run: |
+                echo ${{ steps.retrieve-run-id.outputs.run_sha }}
             - name: Download build artifact from triggered workflow
-              uses: dawidd6/action-download-artifact@v2
+              uses: dawidd6/action-download-artifact@v7
               with:
-                run_id: ${{ steps.retrieve-run-id.outputs.run_id }}
+                commit: ${{ steps.retrieve-run-id.outputs.run_sha }}
+                workflow: release.yml
                 name: binaries
                 path: target/release
-                search_artifacts: true
+                allow_forks: true
```

Comment:
Why do you need to have `allow_forks: true`?

Reply:
I read the guy recommends it

Reply:
besides we were running with allow_forks already, it's just in the new version the default is false

Reply:
If we do not need it, let's set it to false as making it true explicitly allows this attack: https://github.com/dawidd6/action-download-artifact/security/advisories/GHSA-5xr6-xhww-33m4

Which is the reason why it is by default false.

Reply:
fixed